docs/docs/user/references/editor-integration.md

```diff
@@ -1,16 +1,19 @@
 # Editor integration
 
-KPOps provides JSON schemas that enable autocompletion and validation for some of the files that the user must work with.
+KPOps provides JSON schemas that enable autocompletion and validation for some of
+the files that the user must work with.
 
 ## Supported files
 
-- [`pipeline.yaml`](../components)
-- [`config.yaml`](../config)
+- [`pipeline.yaml`](../../resources/pipeline-components/pipeline.md)
```

Comment:
Is the change intentional? It used to point to the Components section, now it points to an example `pipeline.yaml`

Reply:
Yes it is intentional. I found it a bit confusing that the user clicks on pipeline.yaml and goes to the components overview page. I created an issue where we should add a dedicated section only to define the pipeline.yaml file.

Reply:
I wanted to do that long ago, but never came up with a better layout. Currently the Components section kinda is the pipeline definition explanation section and it is not very intuitive if you know nothing about kpops.

Reply:
Yeah, I know. We can keep still the components section and explain more in depth what each component is, instead of only saying that `componentA` is subclass of `componentB`

Reply:
Agreed :)